src/components/virtualRepeat/demoDeferredLoading/index.html

```diff
@@ -0,0 +1,22 @@
+<div ng-controller="AppCtrl as ctrl">
+  <md-content layout="column">
+    <p>
+       Display a list of 50,000 items that load on demand in a viewport of only 7 rows (height=40px).
+       <br/><br/>
+       This demo shows scroll and rendering performance gains when using <code>md-virtual-repeat</code>;
+       achieved with the dynamic reuse of rows visible in the viewport area. Developers are required to
+       explicitly use <code>md-virtual-repeat-container</code> as a wrapping parent container.
+       <br/><br/>
+       To enable load-on-demand behavior, developers must pass in a custom instance of
+       mdVirtualRepeatModel (see the example's source for more info).
```

Comment:
I'm thinking it would be easier for the user to instead explicitly set some attribute when they want infinite scrolling and then check that the model provided has the appropriate methods. Something like:
```html
<div md-virtual-repeat="item in ctrl.dynamicItems" md-infinite>
  {{item}}
</div>
```

I think most angular users would rather specify an attribute than extend a base class.